federation-js/package.json

```diff
@@ -17,15 +17,13 @@
   "author": "Apollo <opensource@apollographql.com>",
   "license": "MIT",
   "engines": {
-    "node": ">=8"
+    "node": ">=12.13.0 <15.0"
```

Comment:
Should we follow up and see if we can support / expand to 15.x? I'm not yet familiar with the set of breaking changes in v15 so this may be a naive question.

Reply:
I considered it briefly, but I'm not particularly eager to bother with Node.js 15 until it becomes an even-numbered (i.e., more stable API, destined to be LTS) release.  We'll have several months after that happens to start testing and adjusting accordingly.